app/store/persistent-state/state-migrations.ts

```diff
@@ -30,9 +30,24 @@ type PersistentState_6 = {
   galoyAuthToken: string
 }
 
-const migrate6ToCurrent = (state: PersistentState_6): Promise<PersistentState> =>
+type PersistentState_7 = {
+  schemaVersion: 7
+  galoyInstance: GaloyInstanceInput
+  galoyAuthToken: string
+  galoySavedAccounts: string[]
```

Comment:
Perhaps we can be a little more explicit?

```suggestion
  galoyAuthToken: string
  galoyOtherAuthTokens: string[]
```

Reply:
`galoySavedAccounts` will contain the current token as well so maybe we can rename it to `galoyAllAuthTokens` instead?